tests/functional/test_examples.py

```diff
@@ -33,7 +33,7 @@ def test_hrtf():
 
 
 def test_info():
-    info = run([executable, './palace-hrtf.py', WAV], capture_output=True)
+    info = run([executable, './palace-hrtf.py'], capture_output=True)
```

Comment:
Oh no, why?

Reply:
It's supposed to be running `palace-info.py`. I was copying so there was a lot of mistakes. In the current one I think it's fine.